dbt_rpc/rpc/gc.py

```diff
@@ -26,7 +26,7 @@ def __init__(
 
         if settings is None:
             self.settings = GCSettings(
-                maxsize=1000, reapsize=500, auto_reap_age=timedelta(days=30)
+                maxsize=30, reapsize=15, auto_reap_age=timedelta(days=30)
```

Comment:
Feels right to reduce the default. What do you think about parametrizing this (via env vars)? Different users will be running this in different memory environments, and it would give our colleagues a bit more flexibility to fine-tune this in the future

Reply:
I am going to adjust this to configurable, but sound like long term we probably want to make garbage collector `smarter`, like always keep records of last `dbt run` result

Reply:
Agree on auto-reap being hours (and also an env_var) -- I could anticipate us wanting to evaluate the impact of that being something like `hours=2` but I also don't forsee this being smaller than `hours=1` (which is why I'm disinclined to say "let's make this minutes!")